timescaledb/how-to-guides/hyperfunctions/about-hyperfunctions.md

```diff
@@ -0,0 +1,97 @@
+# About Timescale hyperfunctions
+Timescale hyperfunctions are a specialized set of functions that allow you to to
+analyze time-series data. You can use hyperfunctions to analyze anything you
+have stored as time-series data, including IoT devices, IT systems, marketing
+analytics, user behavior, financial metrics, and cryptocurrency.
+
+Hyperfunctions allows you to perform critical time-series queries quickly,
+analyze time-series data, and extract meaningful information. They aim to
+identify, build, and combine all of the functionality SQL needs to perform
+time-series analysis into a single extension.
+
+Some hyperfunctions are included in the default TimescaleDB product. For
+additional hyperfunctions, you need to install the
+[Timescale Toolkit][install-toolkit] PostgreSQL extension.
+
+|Hyperfunction family|Types|API Calls|Included by default|Toolkit required|
+|-|-|-|-|-|
+|Approximate count distincts|Hyperloglog|`hyperloglog`|❌|✅|
+|||`rollup`|❌|✅|
+|||`distinct_count`|❌|✅|
+|||`stderror`|❌|✅|
+|Statistical aggregates|Statistical functions|`average`|✅|❌|
```

Comment:
These are missing stats_agg, rollup, and rolling and are not available in normal timescaledb, they're all toolkit. Though many of htese are the same functions that are available in normal postgresql except that we have a different calling convention so I might call this, Statistical Aggregates via `stats_agg` or something like that.